lib/src/models/card_model.dart

```diff
@@ -0,0 +1,37 @@
+class CardModel {
+  final String cardId;
+  final String dbfId;
+  final String name;
+  final String cardSet;
+  final String type;
+  final String? text;
+  final int? health;
+  final String playerClass;
+  final String locale;
+  final List<dynamic>? mechanics;
+
+  CardModel(this.cardId,
+      this.dbfId,
+      this.name,
+      this.cardSet,
+      this.type,
+      this.text,
+      this.health,
+      this.playerClass,
+      this.locale,
+      this.mechanics,);
```

Comment:
I think you missed ctrl+alt+l to format code

Reply:
Done